ports/mimxrt10xx/supervisor/port.c

```diff
@@ -398,6 +398,13 @@ void port_interrupt_after_ticks(uint32_t ticks) {
 
 void port_idle_until_interrupt(void) {
     // App note here: https://www.nxp.com/docs/en/application-note/AN12085.pdf
+    // Currently I have disabled the setting into wait mode as this impacts lots of differnt
```

Comment:
```suggestion
    // Currently I have disabled the setting into wait mode as this impacts lots of different
```

Reply:
Rephrase because it won't be disabled, just removed. You could say "low-power sleep code was removed because it turns off power to peripherals.". Your example of PWM problems to mention is good. Then just include the `CLOCK_SetMode()` lines in comments about how it used to be done.